Kajabi/sage-lib
# Sage font updates

## Description
This PR updates the Sage font with new icons, specifically for the Site Settings page:
- at-sign
- bell
- blog
- card-update
- credit-card
- file-money
- home-alt
- kajabi
- lock-alt
- menu-bordered
- newsletter
- phone-toolbar
- plug
- sequences
- speaker
- users-alt
- window-paragraph
- world

## Testing in `sage-lib`
1. Visit http://0.0.0.0:4000/pages/design/icon
2. Ensure that the aforementioned icons appear

## Testing in `kajabi-products`
1. (**LOW**) Currently used icons did not change but ensure that icons appear correctly within the app:
   - [ ] Products
   - [ ] Offers
   - [ ] Affiliates
   - [ ] Marketing
